sourcemash/categorize.py

```diff
@@ -1,7 +1,9 @@
 from collections import Counter
 from string import punctuation
 
-def categorizeItem(title, text, category_dict):
+def categorize_item(title, text, category_dict):
```

Comment:
But category_dict exists for the sole purpose of categorize_item, so it's weird for it to be passed in from scraper. Maybe scraper shouldn't be the class and we should extract the "class" stuff (i.e. the category_dict) to a Categorizer class.